reference/checkouts.sf.yml

```diff
@@ -2407,6 +2407,7 @@ definitions:
   checkout_Full:
```

Comment:
We may want to use the cart_Full model instead of checkoutCart. cart_Full has references to other models making it easier to keep everything linked together.

Reply:
checkouts_Resp's cart object should reference cart_Full

Reply:
Do all of the Storefront Checkouts `lineItems` have the `isTaxable` field? Is that the correct filed name? (Asking since the Jira ticket mentioned `taxable` vs `is_taxable`.)

Reply:
updated taxable vs is_taxable
 is_taxable is used for physical, and digital line items. And taxable is used for gift certificates.